l1-contracts/src.ts/deploy-test-process.ts

```diff
@@ -170,16 +250,33 @@ class EraDeployer extends Deployer {
       maxL2GasPerBatch: SYSTEM_CONFIG.priorityTxMaxGasPerBatch,
       minimalL2GasPrice: SYSTEM_CONFIG.priorityTxMinimalGasPrice,
     };
-    const storedBatchZero = await this.stateTransitionManagerContract(this.deployWallet).storedBatchZero();
+    const storedBatchZero = ethers.utils.keccak256(
+      new ethers.utils.AbiCoder().encode(
+        ["tuple(uint64 a, bytes32 b, uint64 c, uint256 d, bytes32 e, bytes32 f, uint256 g, bytes32 h)"],
```

Comment:
a bit cleaner would be to either use fully named params or encode an array of types with array of values to avoid these `a`,`b`, etc name

Reply:
I added the proper names